cpp/src/arrow/ipc/reader.cc

```diff
@@ -1053,7 +1053,7 @@ class RecordBatchFileReaderImpl : public RecordBatchFileReader {
         file_->ReadAt(footer_offset_ - footer_length - file_end_size, footer_length));
 
     auto data = footer_buffer_->data();
-    flatbuffers::Verifier verifier(data, footer_buffer_->size(), 128);
+    flatbuffers::Verifier verifier(data, footer_buffer_->size(), 128, UINT_MAX);
```

Comment:
please still add the parameter comment if possible.

Reply:
Like this?
```c++
// Increase max_tables from default 100000 to UINT_MAX when verfying a flatbuffer.
flatbuffers::Verifier verifier(data, footer_buffer_->size(), 128, UINT_MAX);
```

Reply:
No, like this:
```c++

flatbuffers::Verifier verifier(data, footer_buffer_->size(), /*max_depth=*/128, /*max_tables=*/UINT_MAX);
```

Reply:
done